rebasehelper/checkers/rpmdiff_tool.py

```diff
@@ -142,7 +192,7 @@ def run_check(cls, results_dir):
             results_dict = cls._analyze_logs(output, results_dict)
 
         results_dict = cls.update_added_removed(results_dict)
-        results_dict = dict((k, v) for k, v in six.iteritems(results_dict) if v)
+        cls.results_dict = dict((k, v) for k, v in six.iteritems(results_dict) if v)
```

Comment:
This could be replaced with [dict comprehension](https://www.python.org/dev/peps/pep-0274/#semantics).